src/tests/utils/events.cairo

```diff
@@ -0,0 +1,91 @@
+use core::array::ArrayTrait;
+use core::array::SpanTrait;
+use openzeppelin::upgrades::UpgradeableComponent::Upgraded;
+use openzeppelin::upgrades::UpgradeableComponent;
+use snforge_std::cheatcodes::events::EventFetcher;
+use snforge_std::{spy_events, SpyOn, Event, EventSpy, EventAssertions};
```

Comment:
```suggestion
use snforge_std::{spy_events, SpyOn, EventSpy, EventAssertions};
```

Reply:
We need `Event` in **drop_events_from** and **drop_all_events_from** functions